### PR TITLE
Class 5 OOP concept pull request.

### PR DESCRIPTION
**Pull request by fahimar**
Task 
Need to do all the function work with using class prototype concept.

### CODE REVIEW

File: class-5/class.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Car {
+  constructor(brand) {
+    this.brand = brand;
+  }
+
+  strartEngine() {
```
Comment (tajnur007) [inline class-5/class.js]: I think you have misspelled this function name, did you mean it **`startEngine`**?

File: class-5/prototypal_Inheritance.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// function Dog(name, breed, color) {
```
Comment (tajnur007) [inline class-5/prototypal_Inheritance.js]: You can discard commenting this code block. Here we can define this functional constructor as `DogFunc` and for class constructor as `DogClass`.

File: class-5/class.js (hunk 1 above)
Comment (fahimar) [reply class-5/class.js]: Yes it is **startEngine**, I am solving it .

**Review by tajnur007 — APPROVED** [review]
Good practice Brother.
Approved from my side.

**Comment by fahimar** [comment]
Thank you, boro bhai.

**Pull request merged**